style2.css

```diff
@@ -0,0 +1,404 @@
+
+  .main__content {
+	    margin: 0 0 40px 0;
+	    padding: 0;
+  }
+
+  .main__content-item {
+	    margin: 0 20px;
+  }
+
+  .content {
+    margin: 0;
+  }
+
+  .content__pic {
+	    width: 65px;
+    height: 50px;
+  }
+
+  .content__one-big {
+    font-size: 50px;
+  }
+
+  .content__one-small {
+    font-size: 20px;
+  }
+
+  .content__item-two {
+	    font-size: 12px;
+  }
+
+  .button__border {
+    color: white;
+    background-color: #e51937;
+    padding: 0 50px;
+  }
+
+  .button__out {
+	    color: #e51937;
+  }
+
+  .button__redwheel {
+	    margin-right: 5px;
+	    align-items: center;
+	    display: flex;
+	    justify-content: center;
+  }
+
+  .button__pic {
+	    display: none;
+  }
+
+  .button__text {
+    display: flex;
+    flex-wrap: nowrap;
+    font-size: 16px;
+  }
+
+  section {
+    padding: 0;
+    margin: 0 auto;
+  }
+
+  article {
+    padding: 0;
+    margin: 0 auto;
+  }
+
+  .section__title {
+    font-size: 70px;
+	    text-align: right;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .body {
+    text-align: center;
+    align-items: center;
+  }
+  
+  .main__title {
+	    font-size: 90px;
+	    margin-bottom: 10px;
+  }
+
+  .main__subtitle {
+    font-size: 15px;
+  }
+
+  .main__pic {
+    margin: -50px 0 0 0;
+  }
+	
+  .main__content {
+    align-items: center;
+    margin: 0 0 20px;
+  }
+ 
+  button {
+ 	  padding: 0 30px;
+  }
+  
+  .content__one-big {
+    font-size: 40px;
+  }
+
+  .content__one-small {
+    font-size: 15px;
+  }
+
+  .content__item-two {
+    font-size: 10px;
+  }
+
+  .content__pic {
+    width: 38px;
+    height: 27px;
+  }
+
+  .roadicon {
+    width: 29px;
+    height: 41px;
+  }
+
+  .button__out {
+    padding: 10px;
+  }
+
+  .button__border {
+    padding: 0 40px;
+  }
+
+  .button__text {
+    font-size: 12px;
+  }
+}
+
+@media screen and (max-width: 560px) {
+  .main__title {
+    font-size: 70px;
+  }
+  
+  .main__pic {
+    margin: -40px 0 20px 0;
+  }
+  
+  .header__pic {
+    width: 138px;
+    height: 42px;
+    margin: 10px;
+  }
+}
+
+@media screen and (max-width: 501px) {
+  .button__text {
+    font-size: 12px;
+  }
+
+  .main__pic {
+    margin: 0;
+  }
+
+  .main__title {
+    font-size: 50px;
+  }
+
+  .main__subtitle {
+    font-size: 10px;
+  }
+  
+  .content__one-big {
+    font-size: 27px;
+  }
+
+  .content__one-small {
+    font-size: 15px;
+  }
+
+  .content__item-two {
+    font-szie: 10px;
+  }
+
+  .content__pic {
+    width: 38px;
+    height: 27px;
+  }
+
+  .roadicon {
+    width: 29px;
+    height: 41px;
+  }
+
+  .button__out {
+    padding: 10px;
+  }
+
+  .button__border {
+    padding: 0 40px;
+  }
+ 
+}
+
+@media screen and (max-width: 460px) {
+  .body {
+    text-align: center;
+    align-items: center;
+  }
+
+  .header {
+    padding: 0;
+    margin: 0;
+  }
+  
+  .main__title {
+    font-size: 50px;
+  }
+
+  .main__subtitle {
+    font-size: 10px;
+    text-align: center;
+  }
+
+  .main__content {
+    margin: 10px;	
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)